File: MedLabO.Services/Database/Spol.cs
using System.ComponentModel.DataAnnotations;

namespace MedLabO.Services.Database
{
    public class Spol
    {
        [Key]
        public int SpolID { get; set; }
        [Required]
        public string Kod { get; set; } = null!;
        [Required]
        public string Naziv { get; set; } = null!;
    }
}

File: MedLabO UI/medlabo_mobile/lib/models/spol/spol.dart
import 'package:json_annotation/json_annotation.dart';

part 'spol.g.dart';

@JsonSerializable()
class Spol {
  int? spolID;
  String? kod;
  String? naziv;

  Spol({this.spolID, this.kod, this.naziv});

  factory Spol.fromJson(Map<String, dynamic> json) => _$SpolFromJson(json);

  Map<String, dynamic> toJson() => _$SpolToJson(this);
}
